src/libmpeg2/motion_comp_vis.c checks for ENABLE_VIS therefore we also need to check for it.

--- src/libmpeg2/motion_comp.c
+++ src/libmpeg2/motion_comp.c
@@ -68,7 +68,7 @@
 	mpeg2_mc = mpeg2_mc_altivec;
     } else
 #endif
-#ifdef ARCH_SPARC
+#if defined(ARCH_SPARC) && defined(ENABLE_VIS)
     if (mm_accel & MM_ACCEL_SPARC_VIS) {
 #ifdef LOG
 	fprintf (stderr, "Using VIS for motion compensation\n");
